content/messaging/references/default-properties/index.mdx

```diff
@@ -0,0 +1,495 @@
+| **Data type** | `string`                                |
+
+### welcome_message
+
+Specifies if a welcome message was sent to the Customer.
+
+|               |                                        |
+| ------------- | -------------------------------------- |
+| **Access**    | `r` for Agents and Customers in events |
+| **Data type** | `bool`                                 |
+
+### welcome_text
+
+Contains the welcome message text. Appears only if `"welcome_message": true`.
+
+|               |                                         |
+| ------------- | --------------------------------------- |
+| **Access**    | `rw` for Customers in chats and threads |
+| **Data type** | `string`                                |
+
+</Text>
+<Code>
+<CodeResponse title={'Properties in the lc2 namespace'}>
+
+```json
+{
+  "properties": {
+      "lc2": {
+          "file_safety_confirmation": true,
+          "form_type": "prechat",
+          "greeting_id": 2,
+          "greeting_unique_id": "Q10X0W041P",
+          "prechat_group_id": 0,
+          "welcome_author_id": "bbb67d600796e9f277e360e842418833",
+          "welcome_message": true,
+          "welcome_text": "Welcome! How can I help you today?"
+      }
+  }
+}
+```
+
+</CodeResponse>
+
+</Code>
+</Section>
+
+<Section>
+<Text>
+
+## rating
+
+The properties describing chat rating.
+
+### comment
+
+Contains the text message rating left by the Customer.
+
+|               |                                                                     |
+| ------------- | ------------------------------------------------------------------- |
+| **Access**    | `r` for Agents and `rw` for Customers in chats, threads, and events |
+| **Data type** | `string`                                                            |
+
+### score
+
+Describes chat rating: `0` - thumbs down, `1` - thumbs up.
+
+|               |                                                                                                                         |
+| ------------- | ----------------------------------------------------------------------------------------------------------------------- |
+| **Access**    | <ul><li>`r` for Agents in chats, threads, and events</li><li>`rw` for Customers in chats, threads, and events</li></ul> |
+| **Data type** | `int`                                                                                                                   |
+| **Values**    | `0` `1`                                                                                                                 |
+
+</Text>
+<Code>
+<CodeResponse title={'Properties in the rating namespace'}>
+
+```json
+{
+  "properties": {
+      "rating": {
+          "comment": "Action completed",
+          "score": 0
+      }
+  }
+}
+```
+
+</CodeResponse>
+
+</Code>
+</Section>
+
+<Section>
+<Text>
+
+## routing
+
+### continuous
+
+Specifies if the continuous messaging mode is enabled.
```

Comment:
Hmm I think it'd be nice to underline that this property can be different across different chats.
Messaging Mode is a product feature but any integration can create both continuous and non-continuous chats despite of Messaging Mode (eg. facebook chats are always continuous even if the license doesn't use Messaging Mode feature).